python/tests/benchmark_variable.py

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Any reason this is in the `tests` folder? If not, move it to a new folder `benchmark`?